.circleci/config.yml

```diff
@@ -19,6 +19,10 @@ jobs:
         type: string
     docker:
       - image: cimg/php:<< parameters.php_version >>
+        environment:
+          HTTPBIN_HOST: http://httpbin
```

Comment:
Is this a typo?

```suggestion
          HTTPBIN_HOST: http://httpbin.org
```

Reply:
Also wondering if it should be `https`. I see both are available as described here https://hub.docker.com/r/kennethreitz/httpbin so maybe it doesn't matter?

Reply:
This references the container below, so hits it locally.